### PR TITLE
journald: bump rate limits

### PR DESCRIPTION
**Pull request by dtardon**
Apparently, it is quite common to hit a problem, where systemd-journald would drop messages because service is logging too fast.

With upstream we have agreed that we should bump the related limits to factor of 10.

### CODE REVIEW

File: src/journal/journald-server.c [hunk 1]
```diff
@@ -78,8 +78,8 @@
 #define USER_JOURNALS_MAX 1024
 
 #define DEFAULT_SYNC_INTERVAL_USEC (5*USEC_PER_MINUTE)
-#define DEFAULT_RATE_LIMIT_INTERVAL (30*USEC_PER_SEC)
-#define DEFAULT_RATE_LIMIT_BURST 1000
+#define DEFAULT_RATE_LIMIT_INTERVAL (300*USEC_PER_SEC)
+#define DEFAULT_RATE_LIMIT_BURST 10000
```
Comment (poettering) [inline src/journal/journald-server.c]: hmm, so this changes the rate limit from 1000 per 3s to 10,000 per 5min. Which besides being coarser doesn't actually change the bandwith we allow at all (i.e. both 1000/30 and 10000/300 is 33.3).

I think your intention is to actually increase the bandwidth here, no? Hence, just bump the BURST value, but leave the INTERVAL level the way it is?

Comment (dtardon) [reply src/journal/journald-server.c]: Hm, right, that's the intention...

**Comment by dtardon** [comment]
Pushed an updated version.

**Comment by poettering** [comment]
Please also update the journald.conf(5) man page

**Comment by dtardon** [comment]
Done.

**Pull request merged**